spec/integration_spec.rb

```diff
@@ -14,20 +14,27 @@
   let(:certificate_file_crt) { "certificate.crt" }
   let(:certificate_file_key) { "certificate.key" }
   let(:port) { Flores::Random.integer(1024..65335) }
+  let(:tcp_port) { port + 1 }
   let(:host) { "127.0.0.1" }
   let(:queue) { [] }
 
   before do
     expect(File).to receive(:read).at_least(1).with(certificate_file_crt) { certificate.first.to_s }
     expect(File).to receive(:read).at_least(1).with(certificate_file_key) { certificate.last.to_s }
-    
-    server = Lumberjack::Server.new(:port => port,
+
+    tcp_server = Lumberjack::Server.new(:port => tcp_port, :address => host, :ssl => false)
+
+    ssl_server = Lumberjack::Server.new(:port => port,
                                     :address => host,
                                     :ssl_certificate => certificate_file_crt,
                                     :ssl_key => certificate_file_key)
 
-    @server = Thread.new do
-      server.run { |data| queue << data }
+    @tcp_server = Thread.new do
+      tcp_server.run { |data| queue << data }
+    end
```

Comment:
Since this server is only needed in the context of "using plain tcp connection" I would add a before block in that context to initialize it.